designs/developer/project-45/index.md

```diff
@@ -0,0 +1,58 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Project
+
+Project is a primary navigation item, only available in the Developer Perspective.  
+The secondary masthead includes the project selector, but not the application selector.  
+
+When **all projects** is selected, the list of projects is displayed.  When a project is selected, the project selector will be updated to have that project selected.
+
+When a single project is selected in the project selector, the Project page is shown.
+
+The page header is Project and the Actions menu is available, if the user has access to any actions.
+This page has 3 tabs: Overview, Details and Project Access.  
+
+## Project Actions
+
+The only available action on a Project is Delete Project.
+
+When the Delete Project action is clicked, a confirmation dialog will be issued.  Once the deletion begins, change the Project selector to All Projects, so that the list of all projects is displayed.
```

Comment:
```suggestion
When the Delete Project action is clicked, a confirmation dialog will be issued.  Once the deletion begins, the Project selector will change to All Projects, so that the list of all projects is displayed.
```

Reply:
Updated to match your suggestion @lizsurette